buddyboss/buddyboss-platform
# PROD-730 #3012 - Fix: Delete the forum using Bulk Actions the discussion of that particular forum is not trashed

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #3012 .

### How to test the changes in this Pull Request:

1. Go to Buddyboss forums and 'Delete all forum using Bulk Action'
2. Go to Buddyboss topics and check all topics are moved in Trash relevant to the forums
2. Open network search and try to search for any discussion.
3. See the count and the results appear correct.


### Proof Screenshots or Video
https://vimeo.com/631927345/6c20bcb37f


### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
